types/artifacts.d.ts

```diff
@@ -409,10 +409,26 @@ declare global {
         attributeWidth: string;
         /** The raw height attribute of the image element. CSS images will be set to the empty string. */
         attributeHeight: string;
-        /** The CSS width property of the image element. */
+        /**
+         * The CSS width property of the image element.
+         * @deprecated Use `element?._privateCssSizing.width` instead.
```

Comment:
should these deprecations be `TODO` since we don't actually want anyone using `_privateCssSizing` yet?

Reply:
yes definitely. done.